src/test/resources/org/jabref/logic/exporter/OldOpenOfficeCalcExportFormatContentSingleEntry.xml

```diff
@@ -1,12 +1,16 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<office:document-content xmlns:office="http://openoffice.org/2000/office" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:class="spreadsheet" xmlns:version="1.0" xmlns:style="http://openoffice.org/2000/style" xmlns:table="http://openoffice.org/2000/table" xmlns:text="http://openoffice.org/2000/text">
```

Comment:
Is a reformat here really necessary?

Reply:
Maybe some automatic thing happened here in VSCode or intellij, I just copied the file as it's written out because it was too difficult to see where the thing actually was..